src/presentation/controller/global/Controller.ts

```diff
@@ -31,6 +32,41 @@ class GlobalController extends AbstractController {
     };
   }
 
+  getSignOut(): RequestHandler {
+    return (request: Request, response: Response, next: NextFunction) => {
+      try {
+        const previousPageUrl = this.getPreviousPageUrl(request);
+        const pageType = super.pageType(request.path);
+        const pageRouting = super.getRouting(
+          globalsRouting,
+          pageType,
+          request
+        );
+        pageRouting.previousUrl = previousPageUrl;
+        response.render(super.templateName(pageRouting.currentUrl), {
+          props: pageRouting
+        });
+      } catch (error) {
+        next(error);
+      }
+    };
+  }
+
+  signOutChoice(): RequestHandler {
+    return (request: Request, response: Response, next: NextFunction) => {
+      try {
+        if (request.body["sign_out"] === 'yes') {
+          return response.redirect(ACCOUNTS_SIGN_OUT_URL);
```

Comment:
we also have to remove data from the cache when they sign out (it's in an note after the ACs on teh ticket)

Reply:
I think that not should be an AC itself, otherwise too easy to miss. Will take a look.

Reply:
Added function to clear cache but unable to add a test that could mock and capture the values.